From 1e5cbb159a233c4ebca9e257b994aec7326bcb37 Mon Sep 17 00:00:00 2001
From: Mooffie <mooffie@gmail.com>
Date: Wed, 1 Jul 2015 20:37:42 +0300
Subject: [PATCH] Menu selector: make "User", not "Local", the focused button.

On the menu selector two buttons are shown. Users are more likely to
want to edit their central user menu, so we move the focus there.
---
 src/filemanager/cmd.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/filemanager/cmd.c b/src/filemanager/cmd.c
index d824c3f..7a8fa8b 100644
--- a/src/filemanager/cmd.c
+++ b/src/filemanager/cmd.c
@@ -1011,6 +1011,7 @@ edit_mc_menu_cmd (void)
     vfs_path_t *menufile_vpath;
     int dir = 0;
 
+    query_set_sel (1);
     dir = query_dialog (_("Menu edit"),
                         _("Which menu file do you want to edit?"),
                         D_NORMAL, geteuid ()? 2 : 3, _("&Local"), _("&User"), _("&System Wide"));
-- 
2.1.4

